kubernetes_deploy/dev/app-config.yml

```diff
@@ -0,0 +1,20 @@
+apiVersion: v1
```

Comment:
circleci/deploy.sh `kubectl apply -f kubernetes_deploy/${environment}/app-config.yaml` is expecting extension 'yaml' instead of 'yml'. Ditto to all the other app-config.yml files.

Reply:
brill, good spot. Thats the effect of my blindness. 👍

Reply:
corrected those files names now `yml --> yaml`